examples/finance/bs_ivbp.ipynb

```diff
@@ -0,0 +1,649 @@
ahlmY2NkcTvcugi550TlpFGgaXwYIzqSVPCsmWq0/RO7Rp//SYukGftt9k39vBFg7r/QYrvSXq3fs86NzlXucOd9q32Va73Grf5mb7Dp+07/FJ+wGfdBos9z2+17zLJ+02NzsetzoWt7sp7vUK9PhxHvbSbHqX2PWv0Qw+Ryd4mz5fwBfX8cSPE4ifQIi/h5Q/jpLvYol3cIMclryGJS5hiRkUJSQTKJK0DCl/EVFg36BMJdQibU+flpi03qAf6M9XyrqCZyAYcesCgeGcxaw5jKmE/SKgT5WNnLJeVC0SGOqGIuoNAvQEMaKu4xv2z5JX8X398ZSigm8weQiCCI51TzsGIISpXxYoWcPEgoR8EwxzRRQRp7nrneJoiJw31vX5Yh6laarct/9XEM/uyHyajXm73SaRSPDlL3+Zv/iLv+Av//IvSaVSfPWrX+UP/uAPTt3ub/3WbyGE4F/9q391vCwIAn71V3+VarVKLBbjp37qp/jhD83HZtgwVqAGHEqpkQtQn8SoE6ij/Wu328cGIalUilqtxrVr14YywD3Cy+xxdV44y/MxKD2bhukcBUHAv3n4X7VjRSfFdiccKF5O1rSmEnEZYcVgmx2R+sffpUSNm60wEYgJh3sGVSZnJ9nQmA9MRYssdfTEpu3rg8yF+DR32/dCyyWCTYPBwUJsmjvtcOB92jrzsRkedPXBesfX209n/SyrPT1RTFlJtjVkKCKjRiv2WmSeuqERr8lyPSJirBocCKuROaO1esYq0PT0x6ITbNPWWNxHRIKN/kO8IJyzmbem2dPWwDiUnSvc7/4IS+SwhPPox0YJi7TM0gv2UY8UM/Go3kzg4RNFyATQg6AHdIEOQdA0EiSQ9D398RU4dA0W6oeqlcnsI4GwDISFOD2DIiZL77UAACAASURBVCeI4BlrhhTeqTVMZht1gTkDR8kpfF//vRTBVaTUqz1SXibQEJJDTJxKgsRp1u1yJtTg9tMPvYY0mEcExLQE6Xi7Ih1KJwywcK2fM66jg+/7xhd2nU7nsTKSXC7H1772Nb72ta/heWYzkW9+85v80R/9USjl73d+53f4vd/7Pf7kT/6Ey5cv8+u//uv8w3/4D/n4448Hsqb7eTEmUAMOKeWpF+4oYJQJ1JGj2u7uLj/4wQ+oVqu8++67L1VxeJUYxRQ++GyE47P0bBoDvrl1k3utcACStmLc7erdufqB/q3tYqKoddgrOxlutfSpeAeG/k4LiQofadL6JqwEdwx9pxJKf85noxUeGtLq2obPnzcQKwHsuvrgcy42w71OeB2ATqD/nFqkxnJXvz+pWJx9zaFOqjR1w+fUojUedPSBoY60AGRPsWkvO9MsdU1BqL6BcETEjc2AS84MWwYzioIzw3LXZLBga1NJBYKmv06ATz9o0Q8+JaM2CfbdBm4QVoSSqkjXv4Wu6CZrXed+/zZKTGOJKJaIoISNwialcmwHeyhxBYFABD7t1j6pZJyIiNEPNhGiC3QRQQdoEwRNYtZVOp6+hiduXaLr6Y0VFJfw0acmRtU1XIMhQ0RdxzXUMNnqDTwDYZFixjgmRNqoPoFACT2xOtxuBNNtXqnLYNh/5FUwOuilEMamuk9J+VJvGHtG+RS1qX2u9T8QyMJpWw1v6xQFqtPpEI3qC+JMZSS7u7v8k3/yT/jjP/5jfv3Xf/14eRAE/P7v/z6/9Eu/xFe+8hUA/vRP/5Riscif//mf87Wvfe255j2IGI0oboQxyuTiCKO2j0EQsLW1RaPRYG9vj0KhQCwW44tf/OJ5T+3MMU7h+xSD2rPpvD//eWFSnxaTBb6/dy+0fD5e5G4rrHwoIWl09Klw+UiKtX44lWYxXua+psZICclKV69eTEcn+VEzHJxP2hPcaetrVpQhgJmNVqh39cSu7elVobnYNA87+qJ0XeoewEykZlSFlCHMy1pZlvv6/ckEGZY1jT8Vkq2+XoGrOjOs9vXznrDS7Gvc9xSKrb6eeE7aVTYM8ys6M9S7+sBWCv13XWGxbdheWhXY6OuVnYKzaLZkd2aNDYFTKkfXoKSoR99hL+jiBd0nEvJ6HHhPzNOGre4hKWuF+nspBBlSfhs3mHtExiIo4aCwUNh0PYkQP/VIGQuQeAhcfLdNt+8BNWzLRakugiZC9AFBEJgJC6fUMCm0XPRwTBbwDAqbra6YTSfkDTDYmgsxdYqtuYM0pF/CYZ8zE4S6jDAYSwRi/hT1SSID/XV9+KHT2p5RffvnzesYcNJE4kmcRqBM+Bf/4l/wj/7RP+Knf/qnHyNQd+/eZWVlhZ/5mZ85XhaLxfjJn/xJvvGNb4wJ1BgvH0cmEqOMUSFQzWaTRqPB2toaExMTTE1NkclkANjcNPQWGXKMogL1PPt01j2bXhaG5Rz9aL/O+7v3QsttqbjX0qtPMaWvGbySqPGjZjhAn7Di3GrqSYopmL4Yr/GJJq0vImweGNL6ipEMO60wSSs6Oe4/pxHF9CnESpdeBlBzKiwb0u2U1O/npDXJkqGOKe9k2NcQUgeHDV9Pkqajcyz19CldJl6fkCmWu3q1qBqZp9HVk5OYirCreVRKLLb6+mOXtUpsGKzVS84iqz19wJu2srQNNWeuoV+URLFn6J0VEWm2DKQrqcpsG+zVs9Y8B4YUvax1iV1Dql3OvnRsfuE+YVqRtd9gy2ABn7Pe4MA6MeYpII0IbCasK/TcFZQoIoWNwkIKC4UgIqJ0ghZSVBC4SPoIukh6WETo+0uIwEGIx69lSR7PYGQBNr6v33cAQdtoa65kmcDXXxNK3gCDxThiFowmDwoR3DPOR4osmIwn5HVkoCd0AUmt856rvkQgLxo/z4TTXJ3b7fZztc74oz/6I27dusWf/dmfhcZWVg6/H6VS6bHlpVKJet1k7T9cGBOoAcDYRGJ497Hf7x8H0EopqtUqX/rSl0I3qGEJYJ8Xo1oDddo+DVvPpmFSoP7Nw29ol7+RqvHB/r3Q8lJkgk8Owg9jQcCOqzeVmI
```

Comment:
Maybe make this paragraph a little less 'personal' (i.e. for other readers down the line) and add details of the hardware being used?

Reply:
Done